modules/ossm-uninstalling-service-mesh-operator-control-plane.adoc

```diff
@@ -0,0 +1,164 @@
+//TP1 content. 3 may be removed for GA. Using now to help users distinguish between OSSM 2.x and OSSM 3 as the uninstall instructions for each Operator are different.
+//Possible uninstall procedures for Kiali, OpenShift Service Mesh Console (OSSMC), and other integrations may be added, or linked to, for GA.
+
+Uninstalling {SMProductName} Operator 3 and Istio the control plane from an existing {ocp-product-title} instance requires removing the following:
+
+* `Istio` resource
+* `IstioCNI` resource
+* {SMProductName} Operator 3
+* `istio-system` project
+* `istio-cni` project
+
+Operator Lifecycle Management lets you decide if you want also remove the Istio custom resource definitions (CRDs). See link:https://olm.operatorframework.io/docs/tasks/uninstall-operator/#step-4-deciding-whether-or-not-to-delete-the-crds-and-apiservices[Operator Lifecycle Management]
+
+[NOTE]
+====
+If no resources remain, it is safe to remove the CRD or `APIServices`. Otherwise, you should not delete the type as the wanted resources will be deleted automatically when the CRD or APIService is deleted.
+====
+
+You can uninstall {SMProductName} Operator 3 either by using the {ocp-product-title} web console, or through the command line interface (CLI).
+
+== Uninstall {SMProduct} by using the web console
+
+. Delete the `Istio` resource:
+.. In the {ocp-short-name} web console, click **Operators** -> **Installed Operators**.
+.. Click **Istio** in the **Provided APIs** column.
+.. Click the **Options** menu -> **Delete Istio**.
+.. At the prompt to confirm the action, click **Delete**.
+
+. Delete the `IstioCNI` resource:
+.. In the {ocp-short-name} web console, click **Operators** -> **Installed Operators**.
+.. Click **IstioCNI** in the **Provided APIs** column.
+.. Click the **Options** menu -> **Delete IstioCNI**.
+.. At the prompt to confirm the action, click **Delete**.
+
+. Uninstall {SMProductName} 3 Operator:
+.. In the {ocp-short-name} web console, click **Operators** -> **Installed Operators**.
+.. Locate {SMProductName} 3 Operator.
+.. Click the **Options** menu -> **Uninstall Operator**.
+.. At the prompt to confirm the action, click **Uninstall**.
+
+. Delete the `istio-system` project:
+.. In the {ocp-short-name} web console, click  **Home** -> **Projects**.
+.. Locate the name of the `istio-system` project.
+.. Click the **Options** menu -> **Delete Project**.
+.. At the prompt to confirm the action, enter the name of the project.
+.. Click **Delete**.
+
+. Delete the `istio-cni` project:
+.. In the {ocp-short-name} web console, click  **Home** -> **Projects**.
+.. Locate the name of the `istio-cni` project.
+.. Click the **Options** menu -> **Delete Project**.
+.. At the prompt to confirm the action, enter the name of the project.
+.. Click **Delete**.
+
+If you decide to delete Istio custom resource definitions, then run the following command:
+
+[source, terminal]
+----
+$ oc get crds -oname | grep istio.io | xargs kubectl delete
```

Comment:
This will remove the istio resources but will leave the sail operator resources. Need to grep for both if you want to fully remove the CRDs installed by the operator.
```suggestion
$ oc get crds -oname | grep -e istio.io -e sailoperator.io | xargs kubectl delete
```